posts/progressive-enhancement/index.qmd

```diff
@@ -0,0 +1,57 @@
+---
+title: "Improving Ecosystem Interoperability Iteratively via Progressive Enhancement"
+author:
+  - name: "Hugo Gruson"
+    orcid: "0000-0002-4094-1476"
+date: "2024-07-04"
+categories: [R, interoperability, S3, progressive enhancement, ecosystem, lifecycle, object orientation, DOI]
+format:
+  html: 
+    toc: true
+---
+
+We are continuing on post series on S3 and interoperability.
+We have previously discussed [what makes a good S3 class and how to choose a good parent for it, as well as when to write or not write a custom method](../parent-class).
+We have highlighted in particular how classes inheriting from data.frames can simplify user experience, and reduce developer workload.
```

Comment:
```suggestion
We have highlighted in particular how classes inheriting from data.frames can simplify user experience because of familiarity, and reduce developer workload.
```